pkg/api/graph/graph.go

```diff
@@ -115,6 +115,11 @@ type Interface interface {
 	Edges() []graph.Edge
 }
 
+type NodeDescriber interface {
```

Comment:
@deads2k i need this to build functions that can take either ReplicaSetNode or ReplicationControllerNode. I wonder if we already have this interface somewhere or not as I was not able to locate any.